validation/jinja2/validation/search.html

```diff
@@ -25,49 +25,44 @@
 
 {% block content %}
 <h2> Results for <q>{{ search_term }}</q> </h2>
+    {%  for phrase in phrases %}
+    <div class="card segment-card">
+    {% if phrase.validated %}
+      <div class="card-header-green">
+          &nbsp;
 
-<div class="table-responsive">
-  <table class="table table-striped">
-    <thead>
-      <tr>
-        <th scope="col"> Transcription </th>
-        <th scope="col"> Translation </th>
-        <th scope="col"> Recordings </th>
-        <th scope="col"> Speaker </th>
-      </tr>
-    </thead>
-    {% for phrase in phrases %}
-    <tr>
-      {{ macros.td_transcription_link(user, phrase) }}
-      {{ macros.td_translation(user, phrase) }}
-      <td>
-        <ul class="recordings">
-        {% for recording in phrase.recordings %}
-          <li>
-            <div class="rec-height">
-              {{ macros.player(recording) }}
-            </div>
-          </li>
+      </div>
+    {% else %}
+        <div class="card-header-red">
+          &nbsp;
 
-        {% endfor %}{# for recording in phrase.recordings #}
-        </ul>
-      </td>
-      <td>
-        <ul class="speaker">
-        {% for recording in phrase.recordings %}
-          <li>
-            <div class="rec-height">
-              {{ macros.speaker(recording) }}
-            </div>
-          </li>
+      </div>
+    {% endif %}
+      <div class="card-body">
+        <h3>{{ phrase.transcription }}</h3>
+        <h5>Translation: {{ phrase.translation }}</h5>
+        <table>
+            <th>Recording</th>
+            <th>Speaker</th>
 
-        {% endfor %}{# for recording in phrase.recordings #}
-        </ul>
-      </td>
-    </tr>
-    {% endfor %}{# for phrase in page.phrases#}
-  </table>
+            {% for recording in phrase.recordings %}
+                <tr>
```

Comment:
Idk what happened to the indentation here but it confused me!

```suggestion
            <tr>
```

Reply:
Oof, me neither. I'll fix all the indentation.